### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Nils-Jung**
Added robot_description & robot_state_publisher to car.launch
Added cartographer parameter for easier access
Fixed cartographer scripts

### CODE REVIEW

File: ros_ws/launch/car.launch [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <launch>
     <arg name="joystick_type" default="ps3"/>
 
+    <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find racer_description)/urdf/racer.xacro' use_gpu:=true visualize_lidar:=true laser_sample_count:=1080"/>
```
Comment (marian42) [inline ros_ws/launch/car.launch]: `use_gpu:=true visualize_lidar:=true laser_sample_count:=1080"`
Why are these needed? Shouldn't they be irrelevant outside of Gazebo?

Comment (Nils-Jung) [reply ros_ws/launch/car.launch]: They are but xacro needs _some_ values for those to compile. Maybe it's possible to write some default values into the xacro file, idk.

Comment (Nils-Jung) [reply ros_ws/launch/car.launch]: Fixed it. :P

File: ros_ws/src/simulation/racer_description/urdf/racer.xacro [hunk 2]
```diff
@@ -3,6 +3,10 @@
 <robot name="myrobot" 
   xmlns:xacro="http://www.ros.org/wiki/xacro">
 
+  <xacro:arg name="use_gpu" default="true"/>
+  <xacro:arg name="visualize_lidar" default="false"/>
+  <xacro:arg name="laser_sample_count" default="1080"/>
```
Comment (marian42) [inline ros_ws/src/simulation/racer_description/urdf/racer.xacro]: Since these are only used in `racer_plugins.gazebo`, the default values should be set there.

File: ros_ws/src/simulation/racer_description/urdf/racer.xacro [hunk 3]
```diff
@@ -3,6 +3,10 @@
...
+
   <xacro:include filename="$(find racer_description)/urdf/racer_plugins.gazebo" />
```
Comment (marian42) [inline ros_ws/src/simulation/racer_description/urdf/racer.xacro]: Now that this xacro file is used outside of gazebo, I think it makes sense to make this include optional.
```xml
<xacro:arg name="use_gazebo" default="false"/>
<xacro:if value="$(arg use_gazebo)">
    <xacro:include filename="$(find racer_description)/urdf/racer_plugins.gazebo" />
</xacro:if>
```

Comment (Nils-Jung) [reply ros_ws/src/simulation/racer_description/urdf/racer.xacro]: Does it matter though? The content of racer_plugins is only used by gazebo anyway, making an argument to include that file is pretty much useless.

File: ros_ws/src/simulation/racer_description/urdf/racer.xacro (hunk 2 above)
Comment (Nils-Jung) [reply ros_ws/src/simulation/racer_description/urdf/racer.xacro]: Done.
